### DISCUSSION TITLE
Where can i get the Language Pack?

### DISCUSSION THREAD

**Discussion by luckyju2008** [eclipse-birt/birt#1662] [Q&A] [answered]
The download page show: BIRT Language Pack 4.15.0, but not provide link for download, where can i find it?

**Comment by merks** [comment, marked as answer]
There is no download for the bundles in the language package.   The translation fragments are in the update site.

![image](https://github.com/eclipse-birt/birt/assets/208716/7bedd406-38ce-4e4d-b5e0-b9beb8527389)

So if you use the little toolkit button:

![image](https://github.com/eclipse-birt/birt/assets/208716/44f6ccd6-46bf-490e-b580-6bc4cd523e40)

And go to the plugins folder:

https://download.eclipse.org/justj/?file=birt/updates/release/4.15.0/plugins

You'll find files like this with the corresponding translations:

https://download.eclipse.org/birt/updates/release/4.15.0/plugins/org.eclipse.birt.chart.device.extension.nl1_4.15.0.v202403260939.jar

**Reply by luckyju2008** [reply]
Thanks very much!

**Reply by luckyju2008** [reply]
After copied these language files into eclipse\plugins directory and restart BIRT, but the BIRT ui still in English:
![image](https://github.com/eclipse-birt/birt/assets/168630711/df807a6c-a8dd-461c-9b97-01a09a110fe4)
If need some other settings? I had set Window/Report Design/Preview/Choose your locale
